mm2src/coins/eth.rs

```diff
@@ -6960,6 +6985,20 @@ async fn get_eth_gas_details_from_withdraw_fee(
                 let amount = u256_to_big_decimal(eth_value, eth_coin.decimals).map_mm_err()?;
 
                 return MmError::err(EthGasDetailsErr::AmountTooLow { amount, threshold });
+            } else if error_str.contains("fee cap less than block base fee")
```

Comment:
Did they not send a numeric error code for this?

Reply:
Yes, got it a few times. Most time I don't get it though, might be dependant on the web3 node but not sure.